packages/web-components/src/components/rux-tabs/rux-tabs.tsx

```diff
@@ -36,6 +36,25 @@ export class RuxTabs {
      */
     @Prop() small?: boolean
 
+    // This allows us to hear the selected prop change on tab.
+    // Once we hear it, we need to update the related panels visibilty accordingly.
+    @Listen('ruxtabselected', { target: 'window' })
+    handleTabselected(e: CustomEvent) {
+        const target = e.target as HTMLRuxTabElement
+        //* only change the classlist of panels assoiciated with this rux-tabs component
+        const children = Array.from(this.el.children)
+        if (target.selected && children.includes(target)) {
```

Comment:
@markacianfrani @FMorrison87 The reason the tests were failing is because I didn't filter out the events to be just the ones from children of the current rux-tabs, so if you had more than one rux-tabs on a page, it'd get jumbled together with the events from other comps. Thank god for tests!